source/MapDetailPanel.h

```diff
@@ -57,7 +59,12 @@ class MapDetailPanel : public MapPanel {
 	int governmentY = 0;
 	int tradeY = 0;
 	
+	// Default display scaling for orbits within the currently displayed system.
+	double scale = .03;
```

Comment:
Should this be const? It looks like the value is never changed.

Reply:
MapDetailPanel.cpp L616: `scale *= 115. / maxDistance;`